policy/engine_test.go

```diff
@@ -2,17 +2,37 @@ package policy
 
 import (
 	"context"
+	"io/ioutil"
+	"os"
 	"testing"
 
 	"github.com/open-policy-agent/conftest/parser"
 	"github.com/open-policy-agent/opa/ast"
 )
 
+func TestCapabilities(t *testing.T) {
+	ctx := context.Background()
+	policies := []string{"../examples/kubernetes/policy"}
+	defer os.Remove("capabilities.json")
+
+	_ = ioutil.WriteFile("capabilities.json", []byte("invalid-json-body"), 0600)
+	_, err := Load(ctx, policies, "capabilities.json")
+	if err == nil {
+		t.Errorf("against invalid json: %v", err)
+	}
+
+	_ = ioutil.WriteFile("capabilities.json", []byte("{}"), 0600)
+	_, err = Load(ctx, policies, "capabilities.json")
+	if err == nil {
+		t.Errorf("against a json that restricts usage of all functions: %v", err)
+	}
```

Comment:
When testing multiple conditions for a function, a good approach is to use a table-driven approach to reduce test boilerplate. This pattern is generally considered best practice for Go. Please revise these tests to use a table-driven approach.